### ISSUE TITLE
Issues with Yubikey

### ISSUE THREAD

**Issue by kaincenteno** [pbatard/libwdi#327]
Yubikeys stop working/being detected by the yubikey manager app if not using the HID-compliant fido driver.

**Comment by pbatard** [comment]
I'm tempted to say _"Well, duh!"_

If you use Zadig/libwdi to **replace** the HID driver, then any application that relies on the HID driver to communicate with the device will be unable to do so.

Drivers are **NOT** interchangeable, meaning that you can not expect WinUSB or the other drivers that libwdi/Zadig install to provide the same APIs for communicating with the device. The end result is that you **must** use an application that was tailored to use the specific driver that you have installed. And it's (mostly) an exclusive or, meaning that you cannot use 2 drivers at once (such as both HID and WinUSB at the same time).

In short, if you want to continue using the yubikey manager app then you should not replace the driver.
